src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,59 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+
     public void getStatistic(String fromFileName, String toFileName) {
 
+        writeResultFile(toFileName, readAndCalculatedFile(fromFileName));
+    }
+
+    private String readAndCalculatedFile(String fileName) {
```

Comment:
The method name 'readAndCalculatedFile' does not follow Java naming conventions. It should be a verb or a verb phrase, such as 'readAndCalculateFromFile'.